### DISCUSSION TITLE
Star playlists

### DISCUSSION THREAD

**Discussion by test2a** [gokadzev/Musify#164] [Ideas]
Would it be possible to star playlists?

**Comment by test2a** [comment]
also, a way to get more "suggested playlists" than the ones in the app currently, can't we use open.spotify.com for these things?

**Reply by gokadzev** [reply]
> also, a way to get more "suggested playlists" than the ones in the app currently, can't we use open.spotify.com for these things?

I'm thinking about performance and accuracy, besides implementing the Spotify API, this feature needs to:
- Fetch playlists from Spotify
- Search all playlist song names in YouTube
- Get all playlist song information which are found in YouTube

**Comment by gokadzev** [comment]
> Would it be possible to star playlists?

Hello, yes it's possible. The problem is we have static playlists which make these features less useful

**Reply by test2a** [reply]
@gokadzev i am confused. are we using youtube full or youtube music?

**Reply by gokadzev** [reply]
> @gokadzev i am confused. are we using youtube full or youtube music?

Only full YouTube
